akka-stream/src/main/mima-filters/2.5.8.backwards.excludes

```diff
@@ -45,4 +45,10 @@ ProblemFilters.exclude[MissingClassProblem]("akka.stream.stage.GraphStageLogic$C
 ProblemFilters.exclude[FinalClassProblem]("akka.stream.stage.GraphStageLogic$ConcurrentAsyncCallback$Pending")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffLogic.onCompleteOrFailure")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffLogic.this")
-ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffSource.this")
\ No newline at end of file
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffSource.this")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartSource.onFailuresWithBackoff")
```

Comment:
Please add an empty line with a comment referencing the issue and a minimal description (the above is one single such chunk).

Reply:
Added a comment, should I squash all the commits into the first one or is this done by the CI / the merger?

Reply:
We can do that through the github ui when we merge, thanks!